src/utils/Fields/renderPlaceholder.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```

Comment:
@jmcolella If you are consolidating the select fields, I imagine that this file will be deleted?